packages/docs/src/components/presets-demo.js

```diff
@@ -46,7 +46,7 @@ export default function PresetsDemo() {
           <span>Preset:</span>
           <Select
             id="theme"
-            sx={{ display: 'inline-flex' }}
+            sx={{ display: 'inline-flex', paddingRight: 24 }}
```

Comment:
Good catch here. Let's increase it further though? On my iPhone:

![image](https://user-images.githubusercontent.com/5074763/147692401-5d465f1b-029f-4fce-a9bc-903f4215f42f.jpeg)

Reply:
@lachlanjc , I've added the style to the core `Select` component : [chore: update style for core Select component](https://github.com/system-ui/theme-ui/pull/2058)